arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -126,6 +147,16 @@ def validateXbrlStart(val, parameters=None, *args, **kwargs):
     val.extensionImportedUrls = set()
     val.unconsolidated = any("unconsolidated" in n for n in val.disclosureSystem.names)
     val.consolidated = not val.unconsolidated
+    val.authority = None
+    if parameters:
+        p = parameters.get(qname("authority",noPrefixIsNoNamespace=True))
```

Comment:
Nit: Defaulting get can get rid of the `if p` below `parameters.get(qname("authority", noPrefixisNoNamespace=True), [])`

Reply:
see preceding comment